modules/dag-shared/src/main/scala/org/tessellation/dag/validators/TransactionValidator.scala

```diff
@@ -0,0 +1,208 @@
+package org.tessellation.dag.validators
```

Comment:
Shouldn't we agree to group by type not by feature?
IMO transaction validator should be in ```org.tessellation.dag.transaction```

Reply:
Ok!